mmv1/products/alloydb/User.yaml

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'User'
+self_link: '{{cluster}}/users/{{user_id}}'
+base_url: '{{cluster}}/users'
+create_url: '{{cluster}}/users?userId={{user_id}}'
+update_url: '{{cluster}}/users?userId={{user_id}}'
+update_verb: :POST
```

Comment:
Yes, the API doc says `patch` and ideally it should be `patch`. While working on this Terraform integration, we identified an issue that prevents updating a user's database roles and password at the same time. We are tracking the work in our backlog and it requires a data deployment that may take some time.

So the plan is:
- Go with `post` for now for updateUser
- Address the backend issue and wait for its deployment
- Change the verb to `patch` afterwards

Reply:
Yes, using the "post" functions the same way as updating the resource here. 

Yes, if we would use "patch" now, it will require two patch calls (one for password, one for database roles).

Our plan is to unblock the terraform integration now using "post". Once the backend supports updating multiple fields in one request with "patch" verb, then switch the Terraform definition here.